components/KiboCollectedProduct.vue

```diff
@@ -0,0 +1,217 @@
+              {{ getPurchaseLocation }}
+            </p>
+          </template>
+          <template #description v-bind="{ description }">
+            <p class="sf-radio__description" @click="handleStoreLocatorClick">
+              {{ getPurchaseLocation ? "Change Store" : "Select Store" }}
+            </p>
+          </template>
+        </SfRadio>
+      </div>
+    </div>
+    <div class="kibo-collectedProduct__remove">
+      <SfButton class="kibo-collectedProduct__remove-button" link="/cart">
+        <font-awesome-icon :icon="['fas', 'trash-alt']" />
+      </SfButton>
+    </div>
+  </div>
+</template>
+<script>
+import {
+  SfPrice,
+  SfIcon,
+  SfImage,
+  SfCircleIcon,
+  SfButton,
+  SfQuantitySelector,
+  SfLink,
+  SfProperty,
+  SfAccordion,
+  SfRadio,
+} from "@storefront-ui/vue"
+import useUiState from "../composables/useUiState"
+
+export default defineComponent({
+  name: "KiboCollectedProduct",
+  components: {
+    SfButton,
+    SfIcon,
+    SfImage,
+    SfCircleIcon,
+    SfPrice,
+    SfQuantitySelector,
+    SfLink,
+    SfProperty,
+    SfAccordion,
+    SfRadio,
+  },
+  model: {
+    prop: "qty",
+  },
+  props: {
+    /**
+     * Product image
+     * It should be an url of the product
+     */
+    image: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Product image width, without unit
+     */
+    imageWidth: {
+      type: [String, Number],
+      default: 140,
+    },
+    /**
+     * Product image height, without unit
+     */
+    imageHeight: {
+      type: [String, Number],
+      default: 200,
+    },
+    /**
+     * Product title
+     */
+    title: {
+      type: String,
+      default: "",
+    },
+    /**
+     * Product regular price
+     */
+    regularPrice: {
+      type: [Number, String],
+      default: null,
+    },
+    /**
+     * Product special price
+     */
+    specialPrice: {
+      type: [Number, String],
+      default: null,
+    },
+    /**
+     * Selected quantity
+     */
+    qty: {
+      type: [Number, String],
+      default: 1,
+    },
+    /**
+     * Link to product
+     */
+    link: {
+      type: [String, Object],
+      default: "",
+    },
+    purchaseLocation: {
+      type: String,
+      default: "",
+    },
+    options: {
+      type: Array,
+      // eslint-disable-next-line vue/require-valid-default-prop
+      default: [],
+    },
+  },
+  setup(props, context) {
+    const { toggleStoreLocatorModal } = useUiState()
+
+    const removeHandler = () => {
+      context.emit("click:remove")
+    }
+    const handleStoreLocatorClick = () => {
+      toggleStoreLocatorModal()
+    }
+
+    const componentIs = computed(() => {
+      return props.link ? "SfLink" : "div"
+    })
+
+    const quantity = computed(() => {
+      return typeof props.qty === "string" ? Number(props.qty) : props.qty
+    })
+
+    const getPurchaseLocation = computed(() => {
+      return props.purchaseLocation !== "Select My Store"
```

Comment:
rename 'getPurchaseLocation' it sounds like a fn
pass nothing instead of "Select My Store" if there is no purchase location

Reply:
Done